app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,99 @@
+<div class="usa-overlay"></div>
```

Comment:
i made a partial because i was tired of scrolling around in the application.html.erb file